coverage_point_calculator/src/lib.rs

```diff
@@ -219,27 +230,36 @@ impl CoveragePoints {
 pub enum BoostedHexStatus {
     Eligible,
     WifiLocationScoreBelowThreshold(Decimal),
+    AverageAssertedDistanceOverLimit(Decimal),
     RadioThresholdNotMet,
     ServiceProviderBanned,
 }
 
 impl BoostedHexStatus {
     fn new(
```

Comment:
If the radio is not eligible due to `service_provider_banned` or ` radio_threshold_not_met` should those be returned regardless of `location_trust_multiplier` and `average_asserted_distance_over_limit`?

Reply:
I agree. 
Active reasons for not receiving boosted rewards should take precedence over passive reasons.
updated [#3d7a1cd](https://github.com/helium/oracles/pull/840/commits/3d7a1cd2582d1d44352282930f370c94caa6bb00)